axelor-base/src/main/java/com/axelor/apps/base/exceptions/BaseExceptionMessage.java

```diff
@@ -640,4 +640,7 @@ private BaseExceptionMessage() {}
 
   public static final String IMPORT_CONFIGURATION_CLOSING_MESSAGE = /*$$(*/
       "The import configuration executed successfully." /*)*/;
+
+  public static final String IMPORT_CONFIGURATION_WRONG_FILE_TYPE_MESSAGE = /*$$(*/
+      "The binding file should be only a xml file, the data file should be only a xml or csv file. Please use the AOP accepted file feature." /*)*/;
```

Comment:
```suggestion
      "The binding file should be only an XML file." /*)*/;
```

There are some typos in the message.

Plus, I think it's better to split it into 2 distinct messages.